Header header
int32 selfID
int32 neigID
float64 ranging
float64 snr
int64 dayInfo
int64 timeStamp 